homeassistant/components/vesync/const.py

```diff
@@ -35,5 +35,10 @@
     "Core600S": "Core600S",
     "LAP-C601S-WUS": "Core600S",  # Alt ID Model Core600S
     "LAP-C601S-WUSR": "Core600S",  # Alt ID Model Core600S
-    "LAP-C601S-WEU": "Core600S",  # Alt ID Model Core600S
+    "LAP-C601S-WEU": "Core600S",  # Alt ID Model Core600S,
+    "LAP-V201S-AASR": "Vital200S",
```

Comment:
Can we break out the addition of Vital200S to a separate PR? We prefer only changing code that the library bump requires when bumping a library.